src/components/ebay-toggle-button-group/component.ts

```diff
@@ -13,6 +13,11 @@ interface ToggleButtonGroupInput
     extends Omit<Marko.Input<"span">, `on${string}`> {
     buttons?: Marko.RepeatableAttrTag<Omit<ToggleButtonInput, `on${string}`>>;
     variant?: "checkbox" | "radio" | "radio-toggle";
+    "a11y-text"?: string;
+    "a11y-label-id"?: string;
+    columnsXS?: number;
+    columnsSM?: number;
+    columnsMD?: number;
```

Comment:
We need one for `XL` too since Skin has it as well.

Reply:
Will add